testing/MLDB-1275_melt_procedure.py

```diff
@@ -0,0 +1,96 @@
+
+# This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+
+import json, datetime
+
+
+def assertVal(res, rowName, colName, value):
```

Comment:
Should we reformat that test with python unittest? I thought this was our new standard.